dnn/variational_inference.ipynb

```diff
@@ -7,30 +7,30 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        for nudge in [x/100.0 for x in range(1, max_degrees_deviation*100)]:
</pre><p>I think you want to start at zero here, not one.</p><p> </p><p>Also, you could use the actual pixel size, but using 1/100th of a degree doesnt seem unreasonable as long as the perf isnt terrible.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Starting at zero is the same as checking if the origin is valid, which is done in lines 14/15 before entering the loop